packages/core/src/components/scroll/scroll.tsx

```diff
@@ -82,9 +82,6 @@ export class Scroll {
       }
     }
 
-    detail.directionX = detail.velocityDirectionX = (detail.deltaX > 0 ? 'left' : (detail.deltaX < 0 ? 'right' : null));
-    detail.directionY = detail.velocityDirectionY = (detail.deltaY > 0 ? 'up' : (detail.deltaY < 0 ? 'down' : null));
```

Comment:
I see this code completely unuseful and slow (4 assigns to object + 4 branches)
also no code is using this, less keep things simple, and fast (since this is a hot loop)

If you want to know the direccion just use (deltaX > 0) => right